python/lsst/ts/mtmount/mock/axis_device.py

```diff
@@ -264,11 +265,22 @@ def do_home(self, command):
                 f"must be {AxisMotionState.STOPPED}"
             )
         current_position = self.actuator.path[-1].at(tai).position
-        home_position = current_position + self.home_offset
-        if home_position >= self.actuator.max_position:
+        center_position = (
+            self.cmd_limits.min_position + self.cmd_limits.max_position
+        ) / 2
+        if current_position > center_position:
             home_position = current_position - self.home_offset
-        if home_position < self.actuator.min_position:
-            raise RuntimeError("Bug: can't compute a suitable home position")
+        else:
+            home_position = current_position + self.home_offset
+        if (
+            home_position < self.cmd_limits.min_position
+            or home_position > self.cmd_limits.max_position
+        ):
+            raise RuntimeError(
+                f"Cannot compute a suitable home position; {current_position=}; "
+                f"{self.cmd_limits.min_position=}; {self.cmd_limits.max_position=}; "
+                f"{self.home_offset=}; {home_position=}"
```

Comment:
Not sure I understand to use `{current_position=}` instead of `{current_position}` and vice versa.

Reply:
`f"...{foo=}"` is shorthand for `f"...foo={foo}"`. I'm not sure when support was added to Python, but it has been awhile. It's nice because it reduces repetition and assures that the correct name is printed. It even works with expressions. e.g. f"{len(my_list)=}"